dev/com.ibm.ws.st.liberty.gradle/src/com/ibm/ws/st/liberty/gradle/manager/internal/GradleProjectInspector.java

```diff
@@ -0,0 +1,251 @@
+    }
+    
+    // TODO: This is where we can attach the progress listener and maintain the project connection.
+    private GradleProject getGradleProject(IProgressMonitor monitor)  {
+    		GradleConnector gradleConnector = GradleConnector.newConnector();
+    		System.out.println(project.getLocation().toString());
+    		ProjectConnection connection = gradleConnector.forProjectDirectory(new File(project.getLocation().toString())).connect();
+    		if (connection != null) {
+    			BuildEnvironment buildEnvironment = connection.model(BuildEnvironment.class).get();
+    			ModelBuilder<EclipseProject> model = connection.model(EclipseProject.class);
+    			EclipseProject eclipseProject = model.get();
+    			GradleProject gradleProject = eclipseProject.getGradleProject();
+    			connection.close();
+    			return gradleProject;
+    		}
+    		return null;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public boolean isSupportedModule(IModule module) {
+        if (module == null) {
+            Trace.trace(Trace.INFO, "The gradle project " + project.getName() + " does not map to a supported module type.");
+            return false;
+        }
+
+        String id = module.getModuleType().getId();
+        if ("jst.web".equals(id) || "jst.ear".equals(id)) {
+            return true;
+        }
+
+        Trace.trace(Trace.INFO,
+                    "The gradle project " + project.getName() + " is detected as a module but does not specify a supported packaging type (eg. war packaging type).");
+        return false;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public IModule[] getProjectModules() {
+        IModule[] modules = new IModule[0];
+
+        LibertyGradleConstants.ProjectType projectType = getProjectType();
+        try {
+//            switch (projectType) {
+// TODO: other project types?
+//                case LIBERTY_ASSEMBLY:
+//                    modules = getLibertyAssemblyModules();
+//                    break;
+//                default:
+                    IModule m = ServerUtil.getModule(project);
+                    if (m != null)
+                        modules = new IModule[] { m };
+//            }
+
+            StringBuffer moduleMsg = new StringBuffer("Found the following dependency modules for project " + project.getName() + ": {");
+            for (int i = 0; i < modules.length; i++) {
+                if (i > 0)
+                    moduleMsg.append(", ");
+                moduleMsg.append(modules[i].getName());
+            }
+            moduleMsg.append("}");
+            Trace.trace(Trace.INFO, moduleMsg.toString());
+
+        } catch (Exception e) {
+            Trace.trace(Trace.WARNING,
+                        "Unable to retrieve modules for project " + project.getName() + ".", e);
+        }
+        return modules;
+    }
+
+    public ProjectType getProjectType() {
+        LibertyBuildPluginConfiguration config = getBuildPluginConfiguration(null);
```

Comment:
Remove the code for Liberty Assembly

Reply:
Ok, will also remove it from the enum in LibertyGradleConstants

Reply:
and the constant

Reply:
This will just simply return the standard project type